Sources/MosaiqueAssetsPicker/Views/AssetDetailCell.swift

```diff
@@ -90,6 +138,11 @@ final class AssetDetailCell: UICollectionViewCell, AssetDetailCellBindable {
         spinner?.removeFromSuperview()
         imageView.image = nil
     }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        gradientLayer.frame = assetVideoGradientView.bounds
```

Comment:
If it's possible, I'd like to manage layer's frame in its owner view.
We can create a UIView that uses CAGradientLayer as a base layer.

```swift
class GradientView: UIView {
  override class var layerClass: AnyClass {
    return CAGradientLayer.self
  }
}
```

we will no need to update the frame.